test/test_rack_server.rb

```diff
@@ -20,21 +20,22 @@ def call(env)
         @app.call(env)
       rescue Exception => e
         @exception = e
+STDOUT.syswrite "\n#{e.message}\n"
```

Comment:
Missing indentation?

Reply:
Sorry, that was my debug code.  I'll remove.